harness/determined/pytorch/_pytorch_context.py

```diff
@@ -57,4 +103,344 @@ def get_lr_scheduler(self) -> Optional[pytorch.LRScheduler]:
+                If passed as a string, must be the string "True" or "False".
+            master_weights (bool, optional, default=None):  Optional property override.
+            loss_scale (float or str, optional, default=None):  Optional property override.
+                If passed as a string, must be a string representing a number, e.g., "128.0",
+                or the string "dynamic".
+            cast_model_outputs (torch.dtype, optional, default=None):  Option to ensure that
+                the outputs of your model is always cast to a particular type regardless of
+                ``opt_level``.
+            num_losses (int, optional, default=1):  Option to tell Amp in advance how many
+                losses/backward passes you plan to use.  When used in conjunction with the
+                ``loss_id`` argument to ``amp.scale_loss``, enables Amp to use a different
+                loss scale per loss/backward pass, which can improve stability.
+                If ``num_losses`` is left to 1, Amp will still support multiple losses/backward
+                passes, but use a single global loss scale for all of them.
+            verbosity (int, default=1):  Set to 0 to suppress Amp-related output.
+            min_loss_scale (float, default=None):  Sets a floor for the loss scale values that
+                can be chosen by dynamic loss scaling.  The default value of None means that no
+                floor is imposed. If dynamic loss scaling is not used, `min_loss_scale` is ignored.
+            max_loss_scale (float, default=2.**24):  Sets a ceiling for the loss scale values
+                that can be chosen by dynamic loss scaling.  If dynamic loss scaling is not used,
+                `max_loss_scale` is ignored.
+
+        Returns:
+            Model(s) and optimizer(s) modified according to the ``opt_level``.
+            If  ``optimizers`` args were lists, the corresponding return value will
+            also be a list.
+        """
+        check.false(self._use_amp, "Please only call configure_apex_amp once.")
+        check.eq(
+            num_losses,
+            1,
+            "Only support configure apex amp on distributed training with only one loss",
+        )
+
+        self._use_amp = True
+
+        if self.hvd_config.use:
+            check.eq(
+                self.hvd_config.aggregation_frequency,
+                1,
+                "Mixed precision training (AMP) is not supported with "
+                "aggregation frequency > 1.",
+            )
+
+        check.true(
+            torch.cuda.is_available(),
+            "Mixed precision training (AMP) is supported only on GPU slots.",
+        )
+        check.false(
+            not self.hvd_config.use and self.n_gpus > 1,
+            "To enable mixed precision training (AMP) for parallel training, "
+            'please set `resources["optimized_parallel"] = True`.',
```

Comment:
Wait, I thought we removed `optimized_parallel` entirely?

Reply:
Yeah this error message is very out of date